grama/eval_random.py

```diff
@@ -221,7 +221,7 @@ def eval_sinews(
         >>> df_design >> gr.pt_auto()
         >>> # Vis results
         >>> df_sinew = md >> gr.ev_sinews(df_det="nom")
-        >>> df_sinew >> gr.pt_auto()
+        >>> df_sinew >> gr.plt_auto()
```

Comment:
Pipe-enabled functions use two-letter prefixes; the original code example should be maintained:

```suggestion
        >>> df_sinew >> gr.pt_auto()
```